openhtf/plugs/__init__.py

```diff
@@ -416,15 +420,24 @@ def initialize_plugs(self, plug_types=None):
         if not issubclass(plug_type, BasePlug):
           raise InvalidPlugError(
               'Plug type "%s" is not an instance of BasePlug' % plug_type)
+        if plug_type.logger != _LOG:
+          raise InvalidPlugError(
+              'Do not override "logger" in plugs', plug_type)
```

Comment:
I liked the old wording better; it was less harsh sounding.

Also, I'm personally a fan of end-punctuation on error messages and log lines.